client/components/profilePage/userBookClubs.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Card, Col, Row, Tab, Tabs } from 'react-bootstrap';
+
+export default function UserBookClubs({user}) {
```

Comment:
I think you need to set a default value to this. When refreshing the page, username is briefly undefined, which throws an error if I refresh here.